express-notes-api/index.js

```diff
@@ -0,0 +1,88 @@
+const express = require('express');
+const notesObject = require('./data.json');
+const fs = require('fs');
+const app = express();
+app.listen(3000);
```

Comment:
It's usually good to start `listen`ing after all middleware is set up, and to output a console message indication it's ready along with what port number it's listening on.